src/i18n/en/nav.ts

```diff
@@ -119,6 +131,16 @@ export default [
 	{ text: 'TypeScript Reference', slug: 'guides/typescript', key: 'guides/typescript' },
 	{ text: 'Error Reference', slug: 'reference/error-reference', key: 'reference/error-reference' },
 
+	{ text: 'Astro API Reference', header: true, type: 'api', key: 'api-reference' },
+	{ text: 'The Astro Global', slug: 'reference/api-reference', key: 'reference/api-reference' },
+	{ text: 'astro:actions', slug: 'reference/modules/astro-actions', key: 'reference/modules/astro-actions' },
+	{ text: 'astro:assets', slug: 'reference/modules/astro-assets', key: 'reference/modules/astro-assets' },
+	{ text: 'astro:content', slug: 'reference/modules/astro-content', key: 'reference/modules/astro-content' },
+	{ text: 'astro:i18n', slug: 'reference/modules/astro-i18n', key: 'reference/modules/astro-i18n' },
+	{ text: 'astro:middleware', slug: 'reference/modules/astro-middleware', key: 'reference/modules/astro-middleware' },
+	{ text: 'astro:view-transitions', slug: 'reference/modules/astro-transitions', key: 'reference/modules/astro-transitions' },
```

Comment:
A little renaming... which shouldn't involve links! 😄 
```suggestion
	{ text: 'astro:transitions', slug: 'reference/modules/astro-transitions', key: 'reference/modules/astro-transitions' },
```

Reply:
D'oh! I thought I caught them all! Thank you! 🫡